tests/endpoints/test_cargo_movements_real.py

```diff
@@ -9,8 +9,7 @@
 
 class TestCargoMovementsReal(TestCaseUsingRealAPI):
     def test_default_search(self):
-        results = CargoMovements().search(filter_activity="loading_state")
-        print(len(results))
+        CargoMovements().search(filter_activity="loading_state")
```

Comment:
There should be no param in the default search.

Reply:
This predates my PR, but it's now optional.